server/cli/launch.py

```diff
@@ -183,4 +189,9 @@ def launch(
         f = open(devnull, "w")
         sys.stdout = f
 
-    server.app.run(host=host, debug=debug, port=port, threaded=True)
+    try:
+        server.app.run(host=host, debug=debug, port=port, threaded=True)
+    except OSError as e:
+        if e.errno == errno.EADDRINUSE:
+            raise click.ClickException("Port is in use, please specify an open port using the --port flag.") from e
+        raise e
```

Comment:
I would just do `raise`

I suspect you want the original stacktrace, and I'm not certain this will produce the original stacktrace.

(i tried it out with a toy example, and it does modify the stacktrace)

Reply:
Changed. I didn't notice any issue with my example test, but I didn't check that the stack trace was identical.